Cql/Cql.Packaging/CqlCrosswalk.cs

```diff
@@ -13,9 +13,9 @@
 using Hl7.Fhir.Model;
 namespace Hl7.Cql.Packaging
 {
-    internal class CqlCrosswalk
+    public class CqlCrosswalk

```

Comment:
These classes do feel like internal utility classes to me.  What are they? A kind of mapping from a FHIR type to a CQL type?  Should this also work for other models?

Reply:
Yes, it's a mapping between FHIR and CQL types. We use it to convert our generated outputs to the data model our customers want (FHIR or CQL). I could see it expanding to other data models in the future, but I would think we'd need create a parent and rename this "FhirCqlCrosswalk", which is what it used to be named.